contrail-agent/hooks/vrouter-plugin-relation-joined

```diff
@@ -0,0 +1 @@
+contrail_agent_hooks.py
```

Comment:
this hook is not used in the code. vrouter-agent doesn't pass information to plugin now.